src/routes/(app)/project/[id]/+page.server.ts

```diff
@@ -12,8 +12,10 @@ export async function load({ locals: { getSession, supabase }, params }) {
 		.eq('project_id', params.id)
 		.order('created_at', { ascending: false });
 
-	if (promptReq.error || promptReq.data.length === 0) {
+	if (promptReq.error) {
 		error(500, 'Failed to load prompt');
+	} else if (promptReq.data.length === 0) {
+		redirect(303, `/new/${params.id}/prompt`);
```

Comment:
1. why are we going to print not data here?
2. is this what we want? Nbd but might be confusing if I deleted all examples and then land here.

Reply:
this isn't about data, it checks if there is an initial prompt set.